pkgs/development/compilers/gnu-cim/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchurl
+, fetchzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnu-cim";
+  version = "5.1";
+
+  outputs = ["out" "lib" "man" "info"];
+
+  src = fetchzip {
```

Comment:
Or fetchurl? Saves a bit of storage space (at the cost of a longer unpackPhase).